destinations/v1/destinations_suite_test.go

```diff
@@ -0,0 +1,116 @@
+package destinations
+
+import (
+    "context"
+    "database/sql"
+    "fmt"
+    "log"
+    "testing"
+    "os"
+
+    "github.com/google/uuid"
+
+    "github.com/cloudfoundry/cf-test-helpers/v2/cf"
+    "github.com/cloudfoundry/cf-test-helpers/v2/workflowhelpers"
+    . "github.com/onsi/ginkgo/v2"
+    "github.com/onsi/ginkgo/v2/types"
+    . "github.com/onsi/gomega"
+
+    "github.com/cloudfoundry/cf-performance-tests/helpers"
+)
+
+var testConfig = helpers.NewConfig()
+var testSetup *workflowhelpers.ReproducibleTestSuiteSetup
+var ccdb *sql.DB
+var uaadb *sql.DB
+var ctx context.Context
+var app_guid string
+var app_name string
+var space_guid string
+
+// diego seems to have a limitation here
+// when binding more routes to an app the app does not start, or it will fail during staging already
+const (
+    route_mappings = 800
+)
+
+var _ = BeforeSuite(func() {
+    testSetup = workflowhelpers.NewTestSuiteSetup(&testConfig)
+    testSetup.Setup()
+    ccdb, uaadb, ctx = helpers.OpenDbConnections(testConfig)
+
+    log.Printf("Starting to seed database with testdata...")
+
+    spaceName := testSetup.TestSpace.SpaceName()
+    app_name = fmt.Sprintf("%s-app-%s", testConfig.GetNamePrefix(), uuid.NewString())
+
+    spaceGuids := helpers.GetGUIDs(testSetup.AdminUserContext(), testConfig, fmt.Sprintf("/v3/spaces?names=%s", spaceName))
+    space_guid = spaceGuids[0]
+
+    data := fmt.Sprintf(`{
+                           "name": "%s",
+                           "relationships": {
+                             "space": {
+                               "data": {
+                                 "guid": "%s"
+                               }
+                             }
+                           }
+                         }`, app_name, space_guid)
+
+    exitCode, appCreateBody := helpers.TimeCFCurlReturning(testConfig.LongTimeout, "-X", "POST", "/v3/apps", "-d", data)
+
+    Expect(exitCode).To(Equal(0))
+    Expect(appCreateBody).To(ContainSubstring("201 Created"))
+
+    appCreateResponse := helpers.ParseCreateResponseBody(helpers.RemoveDebugOutput(appCreateBody))
+    app_guid = appCreateResponse.GUID
+
+    helpers.ImportStoredProcedures(ccdb, ctx, testConfig)
+
+    createRouteMappingsStatement := fmt.Sprintf("create_routes_and_route_mappings_for_app('%s', '%s', %d)", app_guid, space_guid, route_mappings)
+    helpers.ExecuteStoredProcedure(ccdb, ctx, createRouteMappingsStatement, testConfig)
+
+    helpers.AnalyzeDB(ccdb, ctx, testConfig)
+    log.Printf("Finished seeding database.")
```

Comment:
Could print a timestamp here (as in the other suites) so that we know how long seeding took.
`fmt.Printf("%v Finished seeding database.\n", time.Now().Format(time.RFC850))`

Reply:
`log` adds a timestamp automatically.